Scripts/Ticket_163723.ps1

```diff
@@ -0,0 +1,45 @@
+﻿
+param (
+    [string] $dbType,
+    [string] $username 
+)
+
+$databaseType = $dbType
+$User = $username
+
+<#
+$file_data = Get-Content -Path "C:\Program Files (x86)\Lansa\Verifier_Connection.dat" -Encoding Byte -Raw
+#>
+
+$file_data = Get-Content "C:\Program Files (x86)\Lansa\Verifier_Connection.dat"
+
+echo($file_data)
+
+<#
+foreach($line in [System.IO.File]::ReadLines("C:\Program Files (x86)\Lansa\Verifier_Connection.dat"))
+{
+    Write-Output $line
+}
+#>
+
+
+$DbTestScript = "C:\Program Files (x86)\AZURESQL\LANSA\VersionControl\Scripts\db-test.ps1"
+
+
+if($file_data -eq $databaseType){
+    echo("Database Type is correct")
+    &  "$DbTestScript" -compile $true -test $true
+}
+
```

Comment:
You are not going to find a match of database type with the lines in verifier_Connection.dat. There is a mapping. See the wiki or the Issue. Its somewhere.

Reply:
The documentation on verifier_connection.dat is here:
https://github.com/lansa/db-regression/wiki#pipeline:~:text=The%20MSSQL%20configuration,execute%20the%20tests

Reply:
Done.